homeassistant/components/streamlabswater/sensor.py

```diff
@@ -0,0 +1,168 @@
+"""Support for Streamlabs Water Monitor Usage."""
+
+from datetime import timedelta
+
+from homeassistant.components.streamlabswater import (
+    DOMAIN as STREAMLABSWATER_DOMAIN)
+from homeassistant.const import VOLUME_GALLONS
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+DEPENDENCIES = ['streamlabswater']
+
+WATER_ICON = 'mdi:water'
+MIN_TIME_BETWEEN_USAGE_UPDATES = timedelta(seconds=60)
+
+NAME_DAILY_USAGE = "Daily Water"
+NAME_MONTHLY_USAGE = "Monthly Water"
+NAME_YEARLY_USAGE = "Yearly Water"
```

Comment:
Snake case please

```suggestion
NAME_YEARLY_USAGE = "yearly_water"
```

Reply:
These are just used for the names which as far as I could tell in other sensors are typically capitalized with spaces (or accepted freeform as config parameters).

Reply:
Correct. For names spaces and capitalization is allowed. Attribute keys should be snake_case underscore